docs/scos/dev/front-end-development/202307.0/oryx/building-components/component-types.md

```diff
@@ -0,0 +1,57 @@
+
+| Atomic Design level | Oryx Component           | Examples                       |
+| ------------------- | ------------------------ | ------------------------------ |
+| Atoms               | Design system components | Button, form element           |
+| Molecules           | Domain components        | Product images, cart entry     |
+| Organisms           | Compositions             | Product carousel, product list |
+| Templates           | Composition references   | Header, footer                 |
+| Pages               | Compositions             | Product page, Cart page        |
+
+## Design system components
+
+The Oryx design system offers a collection of highly reusable components that are essential for building a consistent and visually appealing UI. These components are agnostic of the application's context and serve as the building blocks for domain components.
+
+The design system components ensure a consistent and cohesive visual language across the application. The components do not interact with application logic directly. Because they don't know anything about their surroundings, they're considered fairly "dumb". They are used by domain components, which provide them with properties and dispatch events that can be used by the outer components.
+
+Like any component in Oryx, you can configure and customize design system components or replace them with your own. This enables you to build the required visual language throughout the application. Any reference to a design system component, like `<oryx-button>`, no matter where it is used, is resolved by your custom version.
+
+Design system components are available in the `ui` package: `@spryker-oryx/ui`. They do not depend on any application logic, except for the integration of localized messages.
+
+## Domain components
+
+Oryx functionality is organized in domains. Domain packages contain functional components, also know as _domain components_. For example, all product-related components are organized in the product package.
+
+Domain components leverage the design system components to ensure a consistent UI/UX. The design components are integrated with inputs (properties), and all of their events are handled by domain components.
+
+Domain components integrate with domain services to obtain and update the application state. The services handle the integration with backend APIs and application state management. In a single page application experience, domain components need to support [reactivity](/docs/scos/dev/front-end-development/{{page.version}}/oryx/reactivity/reactivity.html) to ensure the application state is reflected immediately after it is changed. The complexity of reactivity is avoided as much as possible in components by using [signals](/docs/scos/dev/front-end-development/{{page.version}}/oryx/reactivity/signals.html). To keep the components DRY, domains often provide mixins, which components can use.
```

Comment:
Would it make sense to rewrite the last sentence without `DRY`? I feel like it won't contribute to the inclusivity of this content

Reply:
"To not repeat the boilerplate code that is required for each domain component..."

Reply:
rephrased